tally-power-normalization.ipynb

```diff
@@ -0,0 +1,366 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "48bd4b14",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "heating_tally = openmc.Tally()\n",
+    "heating_tally.scores = ['fission-q-recoverable']\n",
+    "model.tallies = [fission_tally, heating_tally]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "4c804fc7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Heating rate = 76235017.82874997 [eV/source]\n"
+     ]
+    }
+   ],
+   "source": [
+    "sp_path = model.run(output=False)\n",
+    "with openmc.StatePoint(sp_path) as sp:\n",
+    "    tally = sp.tallies[heating_tally.id]\n",
+    "    heating_rate_ev = tally.mean.ravel()[0]\n",
+    "    \n",
+    "print(f'Heating rate = {heating_rate_ev} [eV/source]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "310229cd",
+   "metadata": {},
+   "source": [
+    "The OpenMC tally gives us units of \\[eV/source\\] and we need to convert this to \\[J/source\\]:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "e684b4c9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Heating rate = 1.2214222086486662e-11 [J/source]\n"
+     ]
+    }
+   ],
+   "source": [
+    "joule_per_ev = 1.60218e-19\n",
+    "heating_rate = heating_rate_ev * joule_per_ev\n",
+    "print(f'Heating rate = {heating_rate} [J/source]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e9c1002",
+   "metadata": {},
+   "source": [
+    "At this point, let's assume that our spherical reactor has a power of 30 kW. That is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "a7f4ae45",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "power = 30e3  # [J/s]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03d1211b",
+   "metadata": {},
+   "source": [
+    "The tally normalization factor is simply the power in \\[J/s\\] divided by the heating rate in \\[J/source\\]:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "ef03e05b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_per_sec = power / heating_rate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6989d40",
+   "metadata": {},
+   "source": [
+    "Whenever we want our tallied quantities in units of \"per sec\" rather than \"per source neutron\", all we have to do is multiply by the normalization factor. For example, our fission rate would be:"
```

Comment:
It may be helpful to provide all of the unit conversions in one line like so:
![Screenshot from 2023-10-19 17-12-08](https://github.com/openmc-dev/openmc-notebooks/assets/4563941/7a973760-d28f-4c7e-8805-a4193a5a5312)

From:
https://github.com/openmc-dev/openmc-nea-course/blob/main/notebooks/05-tallies-i/Solution/Tallies.ipynb

Reply:
☝🏻 I could take or leave this change. Your call.

Reply:
The equation for normalization already appears in the user's guide and is linked, which is why I hadn't included it here. Also, we don't typically separate [eV/fission] and [n/fission]. Tallying heating gives you [eV/n], so there's no need to explicitly tally fission to determine the normalization.

Reply:
Ah, yeah you're right. That link should suffice. And good point about doing that conversion in one score.